src/corona_hakab_model/generation/matrix_generator.py

```diff
@@ -24,6 +25,36 @@
 from project_structure import OUTPUT_FOLDER
 
 
+class AgentConnections(NamedTuple):
+    daily_connections: set = set()
```

Comment:
Use the correct type, i.e. Set (import it from Typing)
Also, it would be super if you can use Set[float] isntead to further hint about this variable

Reply:
Wandered why it corrected me from Set to set... 
Changed it to Set[int]